packages/turf-line-chunk/index.js

```diff
@@ -0,0 +1,94 @@
+var along = require('@turf/along');
+var lineDistance = require('@turf/line-distance');
+var helpers = require('@turf/helpers');
+var flatten = require('@turf/flatten');
+var linestring = helpers.lineString;
+var point = helpers.point;
+var fc = helpers.featureCollection;
```

Comment:
Call the helpers functions the way they are `fc` = `featureCollection`, `linestring` = `lineString`